### PR TITLE
Release v2.13

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -5,6 +5,42 @@ History
...
 
 
+Parsec v2.13.0-rc1 (2022-10-01)
+-------------------------------
+
+Features
+~~~~~~~~
+
+* Add the webhook sequester service type that pushes encrypted workspace data to
+  3rd party through HTTP A webhook services is a sequester service with an
```
Comment (FirelightFlagboy) [inline HISTORY.rst:15]:
```suggestion
  3rd party through HTTP A webhook services as a sequester service with an
```

File: HISTORY.rst [hunk 2]
```diff
@@ -5,6 +5,42 @@ History
...
+  cloud/issues/3125>`__)
+
+Bugfixes
+~~~~~~~~
+
+* Remove cut status if cut is overriden by a copy  (`#3022
+  <https://github.com/Scille/parsec-cloud/issues/3022>`__)
+* Fixed links in About window not opening  (`#3023
+  <https://github.com/Scille/parsec-cloud/issues/3023>`__)
+* Fix an error that desync postgresql timestamptz type  (`#3071
```
Comment (FirelightFlagboy) [inline HISTORY.rst:37]:
```suggestion
* Fix an error that desync postgresql timestampt type  (`#3071
```

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: HISTORY.rst (hunk 1 above)
Comment (FirelightFlagboy) [reply HISTORY.rst:15]: done

File: HISTORY.rst (hunk 2 above)
Comment (FirelightFlagboy) [reply HISTORY.rst]: done

**Review by FirelightFlagboy — COMMENTED** [review]
Some fragment are waiting to be merge into the changelog for the next RC

**Review by FirelightFlagboy — APPROVED** [review]

**Review by FirelightFlagboy — COMMENTED** [review]
A fragment is still present

**Review by FirelightFlagboy — APPROVED** [review]

**Comment by vxgmichel** [comment]
Follow up on #3340

**Pull request closed without merging**